### ISSUE TITLE
camel case folders & linux 

### ISSUE THREAD

**Issue by parisni** [standardhealth/shr-cli#12]
Hey,

I had an issue while publishing (yarn run ig:publish)
```
Load Content                                                                     (19.0363sec)
Publishing Content Failed: Unable to find file /home/nps/git/shr-cli/./out/fhir/guide/resources/implementationguide/shr.xml (19.0364sec)
                                                                                 (19.0364sec)
Use -? to get command line help                                                  (19.0364sec)
                                                                                 (19.0364sec)
Stack Dump (for debugging):                                                      (19.0364sec)
java.lang.Exception: Unable to find file /home/nps/git/shr-cli/./out/fhir/guide/resources/implementationguide/shr.xml
```

```
ls out/fhir/guide/resources/
CodeSystem  ImplementationGuide  StructureDefinition  ValueSet
```
moving those folders to lower case solved the issue. 

Looks like an unix issue with case sensitive path or something (I am using linux)

Thanks

**Comment by cmoesel** [comment]
Thanks for the report.  Sometimes I forget that my Mac filesystem is case insensitive (it feels like it _should_ be case sensitive).  We'll take a look at this.

**Comment by cmoesel** [comment]
This has been fixed via the [shr-fhir-export 4.2.2 release](https://github.com/standardhealth/shr-fhir-export/releases/tag/v4.2.2) and the [shr-cli 4.3.2 release](https://github.com/standardhealth/shr-cli/releases/tag/v4.3.2).
